Classes/View Controllers/UIViewController+Safari.swift

```diff
@@ -12,8 +12,12 @@ import SafariServices
 extension UIViewController {
 
     func presentSafari(url: URL) {
-		guard let safariViewController = try? SFSafariViewController.configured(with: url) else { return }
-        route_present(to: safariViewController)
+        if UserDefaults.standard.shouldOpenExternalLinksInSafari {
+            UIApplication.shared.open(url)
```

Comment:
Does this not need a `canOpenURL(_ url:)` check?

Reply:
It won't crash but in general, it would be good to check
Maybe put assertion here too?

Reply:
Sounds good